kube/deploy.sh

```diff
@@ -2,9 +2,23 @@
 set -euo pipefail # make bash quit if something weird happens
 
 export KUBE_NAMESPACE=${DRONE_STEP_NAME}
-export VERSION=${VERSION:-latest}
-export KUBE_SERVER=https://kube-api-notprod.notprod.acp.homeoffice.gov.uk
-export KUBE_CERTIFICATE_AUTHORITY="https://raw.githubusercontent.com/UKHomeOffice/acp-ca/master/acp-notprod.crt"
+export KUBE_TOKEN=${KUBE_TOKEN}
+export VERSION=${VERSION}
+
+if [[ ${KUBE_NAMESPACE} == *prod ]]
+then
+    export UPTIME_PERIOD="Mon-Sun 05:00-23:00 Europe/London"
```

Comment:
Might be better to not have an uptime period at all? We probably don't want it spinning down on us halfway through a late-night script run.

I think you can use a downtime period instead of an uptime one to scale the pods to 0 and not automatically start back up again in the morning, which might be better suited for this.